pkgs/applications/misc/bottles/default.nix

```diff
@@ -52,7 +52,12 @@ python3Packages.buildPythonApplication rec {
     dbus-python
     gst-python
     liblarch
-  ] ++ [ steam-run-native ];
+    patool
+  ] ++ [
+    steam-run-native
+    xdg-utils
+    pciutils
```

Comment:
Played around with bottles a bit and seems like it also wants `cabextract` and `wineWowPackages.minimal`(or maybe more) for some operations.

`cabextract` is needed to install wants using the dependencies menu.
The wine packages are needed when switching dxvk or vkd3d in a bottle.

Would be good to add them here.

Reply:
Yeah there might be additional packages missing. Before I picked up nothing worked at all. Just tested a small subset of functionality.
May I ask you to request a change. Since currently I've no real possibility to build that package based on staging.